modules/delegates.js

```diff
@@ -415,7 +415,8 @@ __private.loadDelegates = function (cb) {
 Delegates.prototype.generateDelegateList = function (cb) {
 	library.db.query(sql.delegateList).then(function (result) {
 		__private.delegatesList = result[0].list;
-		return setImmediate(cb);
+		// TODO: wait 0.9.10 backport to 1.0.0 to get delegate list
+		return setImmediate(cb, null, __private.delegatesList);
```

Comment:
Why we return that?

Reply:
BEcause if not `delegateList` does not work, pending backport